Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -213,16 +215,21 @@ static void webKitSettingsSetProperty(GObject* object, guint propId, const GValu
 {
     WebKitSettings* settings = WEBKIT_SETTINGS(object);
 
+#if !ENABLE(2022_GLIB_API)
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+#endif
```

Comment:
The preprocessor guards make this harder to read. I would avoid adding guards unless they are really needed. In this case, suppressing deprecation warnings unnecessarily is harmless, so I would always suppress them.

Reply:
Done.